data/energy_community_model.yml

```diff
@@ -62,7 +81,8 @@ users:  # list of users and the corresponding assets
 
   user2:  # name of the user
     # list of components
-    
+    market_set: 
+      market_type: non_commercial #type of market

```

Comment:
can't we get rid of this double structure?
Cant we do just: market_type: non_commercial and remove the level market_set?
I think that it should work with little changes

Reply:
Resolved